sources/packages/web/src/components/layouts/aest/AESTApplicationSideBar.vue

```diff
@@ -115,11 +142,25 @@ export default defineComponent({
           });
         }
       });
+      const applicationDetail =
+        await ApplicationService.shared.getApplicationDetail(
+          props.applicationId,
+        );
```

Comment:
The API here loads the application dynamic data and could be too costly for the purpose. 

I would say that, we can still display the Link for draft applications as well and update the technical AC instead of calling the `getApplicationDetails()`

When we click on draft application the list would be empty on the page.

Reply:
Updated. The button is still available for draft applications, but the content card will display "Bypass restriction is disabled for draft applications". I'm open to other suggestions.